patches/tModLoader/Terraria/ModLoader/ModDamageClass.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Terraria.ModLoader
+{
+	public abstract class ModDamageClass : ModType
```

Comment:
Definitely rename to DamageClass, now that we have vanilla classes extending from this.